aws/table_aws_ssm_association.go

```diff
@@ -155,9 +156,17 @@ func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Hydrate:     getAwsSSMAssociation,
 			},
+			{
+				Name:        "compliance_items",
+				Description: "List of compliance information for the associated instances.",
```

Comment:
Is this description from existing documentation? If not, is there a list of values we can refer to and put in the description?

Reply:
yes, it is from the api documentation. should we update it ?